### PR TITLE
Ccs 4270 bulk edit

### PR DESCRIPTION
**Pull request by xdavidson**
This PR introduces the following components that allow user to bulk edit metadata:

- bulk operation for metadata management
- bulk operation for confirmation with progress bar
- add dropdown on the pager to allow users to select more than 5 items.

### CODE REVIEW

File: pantheon-bundle/frontend/src/app/searchResults.tsx [hunk 1]
```diff
@@ -20,6 +21,7 @@ import {
 } from "@patternfly/react-core";
 import { SearchIcon } from "@patternfly/react-icons";
 import CheckCircleIcon from "@patternfly/react-icons/dist/js/icons/check-circle-icon"
+// import ExclamationTriangleIcon from "@patternfly/react-icons/dist/js/icons/exclamation-circle-icon"
```
Comment (grantsydney) [inline pantheon-bundle/frontend/src/app/searchResults.tsx:24]: will we be using this component or can we delete this line

File: pantheon-bundle/frontend/src/app/searchResults.tsx [hunk 2]
```diff
@@ -200,11 +230,12 @@ class SearchResults extends Component<IProps, ISearchState> {
...
       }
 
-      backend += "&offset=" + ((this.state.page - 1) * this.state.pageLimit) + "&limit=" + this.state.pageLimit
+      backend += "&offset=" + ((this.state.page - 1) * this.state.itemsPerPage) + "&limit=" + this.state.itemsPerPage
+
       if (!backend.includes("Updated") && !backend.includes("direction")) {
         backend += "&key=Updated&direction=desc"
       }
-
+      // console.log("[buildSearchQuery] backend=>", backend)
```
Comment (grantsydney) [inline pantheon-bundle/frontend/src/app/searchResults.tsx:238]: this can be removed too

File: pantheon-bundle/frontend/src/app/searchResults.tsx [hunk 3]
```diff
@@ -242,9 +273,15 @@ class SearchResults extends Component<IProps, ISearchState> {
         .then(responseJSON => {
           this.setState({ results: responseJSON.results, nextPageRowCount: responseJSON.hasNextPage ? 1 : 0 })
           const data = new Array()
+          // console.log("[doSearch] results=>", this.state.results)
```
Comment (grantsydney) [inline pantheon-bundle/frontend/src/app/searchResults.tsx:276]: same as above

File: pantheon-bundle/frontend/src/app/bulkOperationConfirmation.tsx [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+      <Title headingLevel="h4" size={TitleSizes.md}>
+        <span className="pf-u-pl-sm">{this.props.footer}</span>
+      </Title>
+    );
+
+    return (
+      <React.Fragment>
+        <div className="p2-search__pf-c-alert">
+          <Alert
+            // className="p2-search__pf-c-alert pf-c-alert pf-m-info"
```
Comment (grantsydney) [inline pantheon-bundle/frontend/src/app/bulkOperationConfirmation.tsx:51]: should this Alert have a classname?

Comment (xdavidson) [reply pantheon-bundle/frontend/src/app/bulkOperationConfirmation.tsx:51]: by default the className is pf-c-alert pf-m-info. If attempted to add a new className to it called "p2-search__pf-c-aert" to fix the position issue.  But it didn't work. I had tomove the p2-search__pf-c-alert" class to an outer div.

File: pantheon-bundle/frontend/src/app/searchResults.tsx (hunk 3 above)
Comment (xdavidson) [reply pantheon-bundle/frontend/src/app/searchResults.tsx:276]: done

File: pantheon-bundle/frontend/src/app/searchResults.tsx (hunk 2 above)
Comment (xdavidson) [reply pantheon-bundle/frontend/src/app/searchResults.tsx]: done

File: pantheon-bundle/frontend/src/app/searchResults.tsx (hunk 1 above)
Comment (xdavidson) [reply pantheon-bundle/frontend/src/app/searchResults.tsx]: deleted.

**Review by grantsydney — APPROVED** [review]
lgtm

**Pull request merged**